### PR TITLE
[Bug 16827] dictionary: Fix some (ab|mis)use of enums annotations

### PR DESCRIPTION
**Pull request by peter-b**
Partially fix bug 16827 by modifying several dictionary entries in
order to:
- Remove redundant "one of the following"-type sentences/clauses
- Make formatting more consistent
- Remove "enum" annotations where the value isn't actuall an
  enumeration

### CODE REVIEW

File: docs/dictionary/command/sort-container.lcdoc [hunk 1]
```diff
@@ -35,15 +35,15 @@ sort items of myInfo by word 2 of each -- sort by word 2 of the line
...
-- "datetime": treats the sortKey as a date and/or time
-- "text": sorts using a codepoint by codepoint comparison
-- "binary": sorts using a byte by byte comparison
+direction (enum): If you don't specify a <direction>, the sort is `ascending`.
+- ascending: sorts in ascending order
+- descending: sorts in descending order
+sortType (enum): If you don't specify a <sortType>, the <sortType> is `text`.
+- international: sorts by collation according to the system locale
+- numeric: sorts by number. (Use this form if the sortKey consists of numbers)
+- datetime: treats the sortKey as a date and/or time
```
Comment (livecodeali) [inline docs/dictionary/command/sort-container.lcdoc]: While you're adding angle brackets here, you could do the `<sortKey>` ones too

File: docs/dictionary/function/URLStatus.lcdoc [hunk 2]
```diff
@@ -28,17 +28,17 @@ if URLStatus(myFile) is "error" then get libURLErrorData(myFile)
...
--"uploaded"  : the application has finished uploading the file to the URL
--"error"  : an error occurred and the URL was not transferred
--"timeout"  : the application timed out when attempting to transfer the URL
--empty  : the URL was not loaded, or has been unloaded
+Returns (enum):
+- queued: on hold until a previous request to the same site is completed
+- contacted: the site has been contacted but no data has been sent or received yet
+- requested: the URL has been requested
+- loading, bytesReceived, bytesTotal: the <URL> data is being received
+- uploading,bytesReceived,bytesTotal: the file is being <upload|uploaded> to the URL
```
Comment (livecodeali) [inline docs/dictionary/function/URLStatus.lcdoc]: I'm guessing these should have spaces after the commas too

File: docs/dictionary/function/fontLanguage.lcdoc [hunk 3]
```diff
@@ -20,22 +20,22 @@ the fontLanguage of "Osaka"
...
-- Korean
-- Polish
-- Roman
-- Russian
-- Thai
-- Turkish
-- SimpleChinese
-- Ukrainian
+Returns (enum):
+- ANSI: for English:language fonts
```
Comment (livecodeali) [inline docs/dictionary/function/fontLanguage.lcdoc]: I think this is meant to be `English-language`

**Comment by runrevelanor** [comment]
In the mobileCurrenLocation returns array there are no quotes around the array keys.

**Comment by peter-b** [comment]
@runrevelanor @livecodeali Pull request updated.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 5010148

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 501014883751002aae8c0e6e2821f8df16e5d00c

**Comment by livecode-vulcan** [comment]
😎 test success 501014883751002aae8c0e6e2821f8df16e5d00c
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2139)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2147)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2132)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2148)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2154)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2160)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2170)

**Pull request merged**